### PR TITLE
feat: add `--disable-command` option

### PR DESCRIPTION
**Pull request by Pagliacii**
Specify a command to disable. Can be used multiple times.

### CODE REVIEW

**Comment by yihong0618** [comment]
@frostming can you help to review?

File: README.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ for yihong0618's channel: https://t.me/hyi0618
 2. Get tg token, ask Google or ChatGPT, need get it from [BotFather](https://t.me/BotFather)
 3. python tg.py ${tg_token}
 
+> [!Note]
+> If you want not to use any command, you can use `--disable-command <command>` option to disable it. Can be used multiple times.
```
Comment (yihong0618) [inline README.md:30]: any command -> all of these commands

Comment (yihong0618) [inline README.md:30]: Can be used -> This can be used

File: tg.py [hunk 2]
```diff
@@ -9,12 +9,24 @@ def main():
     # Init args
     parser = argparse.ArgumentParser()
     parser.add_argument("tg_token", help="tg token")
+
+    # Add the 'disable-command' option
```
Comment (yihong0618) [inline tg.py:13]: # Add the 'disable-command' option -> # 'disable-command' option

File: README.md (hunk 1 above)
Comment (Pagliacii) [reply README.md:30]: Maybe "one of these commands" more prefer?

Comment (Pagliacii) [reply README.md:30]: Disable one by one.

Comment (Pagliacii) [reply README.md:30]: Gotcha

**Review by frostming — COMMENTED** [review]
it would be better to show the available commands as choices= argument and the list can be acquired programatically

**Comment by Pagliacii** [comment]
> it would be better to show the available commands as choices= argument and the list can be acquired(获得) programatically最好将可用命令显示为“choices=”参数，并且可以通过编程方式获取列表。关闭

Nice, I will add an available_commands function to gain all commands.

File: handlers/__init__.py [hunk 3]
```diff
@@ -80,3 +80,13 @@ def load_handlers(bot: TeleBot, disable_commands: list[str]) -> None:
     if all_commands:
         bot.set_my_commands(all_commands)
         print("Setting commands done.")
+
+
+def available_commands() -> list[str]:
```
Comment (yihong0618) [inline handlers/__init__.py:85]: available_commands() -> list_available_commands() maybe better?

**Review by yihong0618 — APPROVED** [review]
LGTM

File: handlers/__init__.py (hunk 3 above)
Comment (Pagliacii) [reply handlers/__init__.py]: done

**Pull request merged**